pandas/tseries/index.py

```diff
@@ -225,6 +225,14 @@ def __new__(cls, data=None,
                 verify_integrity=True, normalize=False,
                 closed=None, ambiguous='raise', dtype=None, **kwargs):
 
+        if isinstance(data, Index):
```

Comment:
put a comment why you are using this. maybe call this ``ref_to_data`` or something